src/parsing/binast-serialize-visitor.h

```diff
@@ -159,84 +158,83 @@ void BinAstSerializeVisitor::SerializeStringTable(const AstConsString* function_
   DCHECK(current_index == num_entries + 1);
 }
 
-void BinAstSerializeVisitor::SerializeAst(BinAstNode* root) {
-  // TODO(binast): Remove (they currently make output slightly easier to read)
-  static std::mutex global_lock;
-  std::lock_guard<std::mutex> lock(global_lock);
```

Comment:
(i could just leave this commented out or otherwise not submit this change if this is still helpful for the output for you)

Reply:
Nah, that's okay. If I need it again I can re-add it.